openassessment/xblock/defaults.py

```diff
@@ -130,6 +130,9 @@
     "due": DEFAULT_DUE,
     "must_grade": 5,
     "must_be_graded_by": 3,
+    "enable_flexible_grading": True,
```

Comment:
We'll have to ask upstream to see if this is the desired behavior. Ok to keep for know though, but not to break backwards functionality you'll need to set this to `false`.

Reply:
The implementation is backward compatible. If ``enable_flexible_grading`` is ``None`` it has been assumed as ``False``. For new ones, it will be default ``True``.